### PR TITLE
Update `jupyterlite-core`

### PR DESCRIPTION
**Pull request by jtpio**
Follow-up to https://github.com/jupyterlite/jupyterlite/pull/998

### CODE REVIEW

File: jupyterlite_pyodide_kernel/addons/_base.py [hunk 1]
```diff
@@ -1,94 +0,0 @@
-"""common addon features for ``jupyterlite-pyodide-kernel``
```
Comment (bollwyvl) [inline jupyterlite_pyodide_kernel/addons/_base.py:1]: was intending on using this to extract more common things from the two plugins, aside from `pyodide_settings`: `piplite` will need to be more aware of `pyodide` stuff for #14.

Comment (jtpio) [reply jupyterlite_pyodide_kernel/addons/_base.py:1]: Sure. I can put the PR back to draft and let you pick it up if you want.

Comment (bollwyvl) [reply jupyterlite_pyodide_kernel/addons/_base.py:1]: :blush: i'll try and work on porting https://github.com/jupyterlite/jupyterlite/pull/979 and see what turns up

**Comment by jtpio** [comment]
More happening in https://github.com/jupyterlite/pyodide-kernel/pull/30 and #27.

**Pull request closed without merging**